packages/flutter/lib/src/widgets/text.dart

```diff
@@ -636,6 +640,16 @@ class Text extends StatelessWidget {
   /// (semi-transparent grey).
   final Color? selectionColor;
 
+  /// Controls how tall the selection highlight boxes are computed to be.
```

Comment:
consider making a template for these doc snippets since they seemed to be used in various places (SelectableText/TextField/EditableText).